folioclient/FolioClient.py

```diff
@@ -161,18 +168,69 @@ def authority_source_files(self):
             )
         )
 
+    @property
+    def okapi_headers(self):
+        """Property that returns okapi headers with the current valid Okapi token"""
+        headers = {
+            "x-okapi-token": self.okapi_token,
+        }
+        headers.update(self.base_headers)
+        return headers
+
+    @property
+    def okapi_token(self):
+        """Property that attempts to return a valid Okapi token, refreshing if needed"""
+        if datetime.now(tz.utc) > (self.okapi_token_expires - timedelta(seconds=self.okapi_token_duration.total_seconds() * .2)):
```

Comment:
Would it make sense to make the `.2` multiplier into a class or instance variable, allowing tweaking in some unforeseen environmental circumstances, like a shorter Okapi token TTL than usual?

Reply:
I like it.